### PR TITLE
Api/archive update

### PR DESCRIPTION
**Pull request by shalper**
## Adds archiving public API endpoints
https://deploy-preview-609--logz-docs.netlify.app/api/#tag/Archive-logs

### CODE REVIEW

File: _source/api/logzio-public-api.yml [hunk 1]
```diff
@@ -640,6 +645,146 @@ paths:
...
+      tags:
+        - Archive logs
+      operationId: getSettingsForAccount
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: successful operation
+          schema:
+            $ref: '#/definitions/ArchiveSettingsResponse'
```
Comment (idohalevi) [inline _source/api/logzio-public-api.yml:662]: @shalper where is `#/definitions/ArchiveSettingsResponse` defined?

File: _source/api/logzio-public-api.yml [hunk 2]
```diff
@@ -3138,3 +3283,100 @@ definitions:
...
+  ArchiveSettings:
+    type: object
+    properties:
+      storageType:
+        type: string
+        enum:
+          - S3
+          - BLOB
+        description: Specifies which settings are applicable - AWS S3 bucket or Azure Blob storage.
+        default: none
```
Comment (idohalevi) [inline _source/api/logzio-public-api.yml:3297]: @shalper In the post request, I think this is required. @alexserdukov true?

File: _source/api/logzio-public-api.yml [hunk 3]
```diff
@@ -3138,3 +3283,100 @@ definitions:
...
+        example: S3
+      enabled:
+        type: boolean
+        description: If `true`, archiving is currently enabled.
+        default: true
+        example: true
+      compressed:
+        type: boolean
+        description: If `true`, logs are compressed before they are archived.
+        default: true
```
Comment (idohalevi) [inline _source/api/logzio-public-api.yml:3307]: @alexserdukov I was sure we always archive compressed...isn't that the case? if so why do we give an option to the customer to change it

File: _source/api/logzio-public-api.yml (hunk 1 above)
Comment (shalper) [reply _source/api/logzio-public-api.yml:662]: @idohalevi The definition begins on line 3315.

**Pull request merged**